src/components/OSCALControlProse.js

```diff
@@ -488,7 +486,7 @@ export function OSCALReplacedProseWithByComponentParameterValue(props) {
           <Link underline="hover" href={`#${props.label}`}>
             {props.label}
           </Link>
-        </StyledTooltip>
+        </StyledTooltip>{" "}
```

Comment:
This fixes the issue with the label being "squished" up against the actual text. Same for the changes above where `{props.label}` and `{prose}` are now on the same line.